content/people/peymant.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Peyman Tehrani
+role: student-phd
+status: Ph.D. Student
+yearjoined: 2017
+portrait: peymant.jpg

```

Comment:
Please crop the portrait photo more to make it similar in proportions
and face visibility to everybody else’s, e.g.:

![panteater](https://user-images.githubusercontent.com/275384/41501258-f6170226-7155-11e8-8ada-51cfdc8ca9b2.jpg)

Reply:
After you are done with cropping, you can safely remove the original
photo (in the ‘peymant.jpeg’ file): it is now stored in the commit
history and can be retrieved from there if we will ever need it again.